### DISCUSSION TITLE
modpack/kubejs/1.20.1/KubeJSCourse/KubeJSAdvanced/EventExamples/BlockRightClickedEvent

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#98] [Announcements]
# modpack/kubejs/1.20.1/KubeJSCourse/KubeJSAdvanced/EventExamples/BlockRightClickedEvent

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/modpack/kubejs/1.20.1/KubeJSCourse/KubeJSAdvanced/EventExamples/BlockRightClickedEvent

**Comment by qoorur** [comment]
1.20.1的kube je下 好像没有 event.hand 的属性了。
-BlockLeftClickedEventJS 构造函数里有 hand
-但没有 hand property 和 get hand 的函数。
有点不知道怎么解决 点击两次的问题了。

**Reply by M1hono** [reply]
点击两次的问题还是好解决的，写最简单的那种[防抖](https://zhuanlan.zhihu.com/p/268012169)就可以。
经测试只有LeftClickedEventJS没有，其余都是有的，lat忘记写getter了。

**Comment by TToNull** [comment]
这个示例好像有点问题吧，给斧子耐久赋值会赋值成负的吧O.o
